### PR TITLE
Fixed some pt-br misspells

### PR DESCRIPTION
**Pull request by valtlfelipe**
I still want to check the texts in `_data`. Will let you know when it's ready to merge.

### CODE REVIEW

File: index-pt-br.md [hunk 1]
```diff
@@ -20,33 +20,33 @@ Administradores de DNS
 ======================
 Para começar a entender como está sua conformidade com o EDNS, recomendamos que você utilize o formulário acima que irá produzir um relatório simplificado para o domínio inteiro.
 
-É também possível testar seus servidores de DNS diretamente utilizando a ferramenta [ednscomp](https://ednscomp.isc.org/ednscomp) que mostra um relatório técnico detalhado. Entre com o nome da zona hospedada nos seus servidores de DNS no campo `zone name` e clique no botão `Submit`.
+É também possível testar seus servidores de DNS diretamente utilizando a ferramenta [ednscomp](https://ednscomp.isc.org/ednscomp) que mostra um relatório técnico detalhado. Coloque o nome da zona hospedada nos seus servidores de DNS no campo `zone name` e clique no botão `Submit`.
 
-O resultado dos testes de [ednscomp](https://ednscomp.isc.org/ednscomp) devem ser uma mensagem em verde dizendo `Tudo Ok`.
+O resultado dos testes de [ednscomp](https://ednscomp.isc.org/ednscomp) deve ser uma mensagem em verde dizendo `All Ok`.
```
Comment (joubertredrat) [inline index-pt-br.md]: Neste caso é bom manter "Tudo OK", porque foi traduzido nos report dos relatórios.

Comment (valtlfelipe) [reply index-pt-br.md]: Mas se você ver, essa linha é referente ao teste no site que está indicado ali (https://ednscomp.isc.org/ednscomp), e lá não está traduzido.

**Comment by valtlfelipe** [comment]
It would be nice that #74 is merged so that I can fix the texts

File: index-pt-br.md (hunk 1 above)
Comment (joubertredrat) [reply index-pt-br.md]: Ah, entendi o ponto. Realmente não está traduzido no report, só no site.

**Comment by pspacek** [comment]
Is this still relevant? Or is it obsolete/should be closed without merging?

**Comment by valtlfelipe** [comment]
@pspacek Sorry for the delay, I hadn't much time this last week. Will take a look at the other translations as soon ass possible.

We can also merge this, and I open a new PR for the `_data` texts. What do you think?

**Comment by valtlfelipe** [comment]
You may merge this, since I didn't got time to check the `_data` translations. Ass soon as I can take a look at it, I open a new PR 😉

**Comment by pspacek** [comment]
Thanks!

**Pull request merged**